sim/midas/src/main/cc/Makefile

```diff
@@ -61,7 +61,7 @@ $(OUT_DIR)/$(DRIVER_NAME)-$(PLATFORM): $(design_h) $(lib) $(DRIVER) $(driver_h)
 $(PLATFORM): $(OUT_DIR)/$(DRIVER_NAME)-$(PLATFORM)
 
 # Sources for building MIDAS-level simulators. Must be defined before sources VCS/Verilator Makefrags
-override CXXFLAGS += -include $(design_h)
+override CXXFLAGS += -std=c++17 -include $(design_h)
```

Comment:
Note i switched to C++17 here:
- to harmonize it across all builds
- because it was previously requested
- i wanted to use `inline` on static members. 

It might be out of scope for this change and i can kick it down the road.